rend3/src/managers/skeleton.rs

```diff
@@ -78,30 +78,43 @@ impl SkeletonManager {
         handle: &SkeletonHandle,
         skeleton: Skeleton,
     ) {
-        let internal_mesh = mesh_manager.internal_data_mut(skeleton.mesh.get_raw());
-
-        assert_eq!(
-            internal_mesh.num_joints as usize,
-            skeleton.joint_matrices.len(),
-            "Created a skeleton with an incorrect number of joints. \
-            The mesh has {} joints, but {} joint matrices were provided.",
-            internal_mesh.num_joints as usize,
+        let internal_mesh = mesh_manager.internal_data(skeleton.mesh.get_raw());
+        let num_joints = internal_mesh.num_joints;
```

Comment:
```suggestion
        let num_joints = internal_mesh.num_joints as usize;
```

Cast up here to remove multiple casts down below.